app/javascript/tpi/controllers/index.js

```diff
@@ -0,0 +1,9 @@
+// Load all the controllers within this directory and all subdirectories.
+// Controller files must be named *_controller.js.
+
+import { Application } from "stimulus";
```

Comment:
Maybe we can drop this stuff as well, yeah?

Reply:
Right, I think there will be controllers eventually, so this one can stay.

Reply:
What for?

Reply:
tooltips for sure.